app-modules/service-management/src/Models/ServiceMonitoringTarget.php

```diff
@@ -66,6 +70,14 @@ class ServiceMonitoringTarget extends BaseModel implements Auditable
         'frequency' => ServiceMonitoringFrequency::class,
     ];
 
+    /**
+     * @return HasOne<HistoricalServiceMonitoring, $this>
+     */
+    public function history(): HasOne
+    {
+        return $this->hasOne(HistoricalServiceMonitoring::class);
```

Comment:
This should probably be a `hasMany` as this will be running every hour/day and storing a history record for each invocation.

And then at that point it may make more sense for this to be plural, `histories`.